app/views/users/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <% provide(:title, @user.first_name) %>
-<div class="row">
-  <aside class="col-md-4">
```

Comment:
Are we using bootstrap? This class name rings the bell, or are you hoping that our grid system caters for these classes?